v4_MIGRATION_GUIDE.md

```diff
@@ -103,7 +103,7 @@ Full implementation examples can be found in the payment method feature sections
 
 ### Handling Cancellation 
 
-All user cancellations will be returned as a `BraintreeException` with an error message indicating user cancellation to the callback of the invoked method. 
+All user cancellations will be returned as a `UserCanceledException` with an error message indicating user cancellation to the callback of the invoked method. 
```

Comment:
Take it or leave it suggestion! This sentence feels slightly confusing/wordy to me right now.

```suggestion
When the customer cancels out of a payment flow, a `UserCanceledException` will be returned in the callback of the invoked method.
```